### PR TITLE
Switch to official antlr3

### PR DESCRIPTION
**Pull request by devpow112**
This switched to the official antlr3 repository on github and patches it during cmake config.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -52,3 +52,6 @@
 	branch = master
 	ignore = dirty
 	shallow = true
+[submodule "antlr3"]
+	path = externals/antlr3
+	url = https://github.com/antlr/antlr3
```
Comment (devpow112) [inline .gitmodules:57]: Need to add additional properties

File: externals/CMakeLists.txt [hunk 2]
```diff
@@ -1,5 +1,34 @@
 set(WN_ADDITIONAL_TRACY_OPTIONS)
 
+find_package(Git)
+
+if (GIT_FOUND)
```
Comment (devpow112) [inline externals/CMakeLists.txt:5]: This can be generalized if we need to add more patches in the future

**Comment by devpow112** [comment]
Need to add license file

File: externals/CMakeLists.txt [hunk 3]
```diff
@@ -1,5 +1,34 @@
 set(WN_ADDITIONAL_TRACY_OPTIONS)
 
+find_package(Git)
```
Comment (devpow112) [inline externals/CMakeLists.txt:3]: Change to be required package

**Review by AWoloszyn — APPROVED** [review]

**Pull request merged**